test/typing/plain_files/orm/session.py

```diff
@@ -97,6 +97,9 @@ class Address(Base):
         User.id
     ).offset(User.id)
 
+    # EXPECTED_TYPE: Optional[User]
+    reveal_type(sess.query(User).get(1))
```

Comment:
Can use `assert_type` from typing_extensions.

Reply:
That would need changes to the mypy test fixtures, but maybe it makes sense?

Reply:
Good point. That would be a separate PR.

Reply:
Indeed. Also myoy would need to print the type in all cases even if the asset is correct